api/tests/csuser/test_cs_user_functions.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding utf-8 -*-
+import mock
+
+from django.test import TestCase
+from api.models.csuser import CSUser
+from api.services import SteamAPIService
+
+
+class CsUserFunctionsTesCase(TestCase):
+
+    def test__str__returns_username(self):
+        #  Act
+        user_for_test = CSUser()
+        user_for_test.steam_username = 'Pepe'
+
+        # Assert
+        self.assertEqual(str(user_for_test), user_for_test.steam_username)
+
+    def test_CSUser_init_csgo_as_none(self):
+        #  Act
+        user_for_test = CSUser()
+        # Assert
+        self.assertIsNone(user_for_test.csgo_info)
```

Comment:
What are you testing here exactly? The `python` behavior?

Reply:
Maybe something usefull... I'm testing that the constructor works and csgo_info starts as None. I did something like that in services.py.

Reply:
There are no constructor, the constructor is the default one in python, this is why I ask you what are you testing? Look at CSUser class.